include/libtorrent/settings_pack.hpp

```diff
@@ -1045,8 +1048,15 @@ namespace libtorrent
 			// torrents.
 			//
 			// ``active_limit`` is a hard limit on the number of active (auto
-			// managed) torrents. This limit also applies to slow torrents.
-			//
+			// managed) torrents. This limit also applies to slow torrents and it
+			// does apply to all announcement protocols as well. If you have a
+			// large amount of seeds, only a limited amount get annonced based on
+			// the seed rank. Use low values if you want to work with
```

Comment:
in general, I would stay away from "you". I think "Use low values if you want to work with" should be something like "Use low values when using".